internal/injector/builtin/generator/main.go

```diff
@@ -49,17 +53,25 @@ func main() {
 	file.HeaderComment("")
 	file.HeaderComment(fmt.Sprintf("Code generated by %q; DO NOT EDIT.", "github.com/datadog/orchestion/internal/injector/builtin/generator "+strings.Join(os.Args[1:], " ")))
 
+	checksum := sha512.New()
 	file.Comment("Aspects is the list of built-in aspects.")
 	file.Var().Id("Aspects").Op("=").Index(jen.Op("...")).Qual("github.com/datadog/orchestrion/internal/injector/aspect", "Aspect").ValuesFunc(func(g *jen.Group) {
 		for _, match := range matches {
+			// Checksum header -- SOH <relative file name> STX
```

Comment:
Does this need a rebase? I thought we already merged the checksum PR into the first pancake